Dockerfile

```diff
@@ -11,7 +17,8 @@ RUN curl -sL https://deb.nodesource.com/setup_8.x | bash - && \
   nodejs \
   postgresql-client \
   default-jre-headless \
-  unison \
+  unison && \
+  apt-get install -y -t stretch-backports libreoffice=1:6.1.5-3~bpo9+1 \
```

Comment:
will this version be always present in backports?

Reply:
If it's not we should probably migrate to a newer Debian version already, right?

Reply:
No, I mean specific number, what if it will be replaced with newer build of the package?

Reply:
I wanted to make sure specific version is installed, but I'll remove the version and just install the latest version on the backports repo.